sass/style.sass

```diff
@@ -13,6 +13,10 @@ $textarea-transition-time: 0.6s
 @import "_form.sass"
 @import "_type.sass"
 
+ul > li > ul
+	margin-bottom: 0
+	margin-top: 0
+
```

Comment:
I think this might have affected the deploy form styles. I'll close until its resolved.